token/js/src/instructions/associatedTokenAccount.ts

```diff
@@ -64,6 +65,31 @@ export function createAssociatedTokenAccountIdempotentInstruction(
     );
 }
 
+/**
+ * Construct a createAssociatedTokenAccountIdempotentInstructionShort instruction
+ *
+ * @param payer                    Payer of the initialization fees
+ * @param owner                    Owner of the new account
+ * @param mint                     Token mint account
+ * @param allowOwnerOffCurve       Allow the owner account to be a PDA (Program Derived Address)
+ * @param programId                SPL Token program account
+ * @param associatedTokenProgramId SPL Associated Token program account
+ *
+ * @return Instruction to add to a transaction
+ */
+export function createAssociatedTokenAccountIdempotentInstructionShort(
```

Comment:
nit: can we rename this to something a bit clearer? maybe `createAssociatedTokenAccountIdempotentInstructionWithDerivation`?